ballerina/modules/charitycheckpdf/client.bal

```diff
@@ -0,0 +1,55 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Are we not adding licensing headers to auto-generated connectors ? @NipunaRanasinghe FYI

Reply:
IINM we can add licensing headers via the openAPI generator tool itself via a flag. @AzeemMuzammil shall we do that?